package.json

```diff
@@ -57,8 +57,11 @@
     "@reduxjs/toolkit": "1.8.0",
     "@talismn/web3react-v6-connector": "6.0.1",
     "@types/react-paginate": "^7.1.1",
+    "@walletconnect/ethereum-provider": "^2.7.5",
+    "@web3-react/abstract-connector": "6.0.7",
     "@web3-react/core": "6.0.9",
-    "@web3-react/walletconnect-connector": "6.2.13",
+    "@web3-react/types": "6.0.7",
+    "@web3modal/standalone": "^2.4.3",
```

Comment:
do we need this package `@web3modal/standalone` ?

Reply:
without this fails to build the package on vercel
https://vercel.com/pangolindex/components/8sEGJUkqQ3AnTEmxJi5dwwX5UVTL

Reply:
yes make sense.